website/sovereign/src/styles/tokens.css

```
@@ -0,0 +1,79 @@
+  /* competitor baseline */
+  --grey: #6b7585;
+
+  /* geometry */
+  --radius: 14px;
+  --max-width: 1140px;
+  --section-rhythm: 120px;
+
+  /* type families */
+  --font-display: 'Fraunces', 'Inter', ui-serif, serif;
+  --font-sans: 'Inter', ui-sans-serif, system-ui, sans-serif;
+  --font-mono: 'Geist Mono', ui-monospace, SFMono-Regular, monospace;
```

Stylelint (17.12.0)

[error] Expected "SFMono-Regular" to be "sfmono-regular" (value-keyword-case)

(value-keyword-case)